modal-js/src/sandbox.ts

```diff
@@ -191,6 +191,24 @@ export class Sandbox {
     }
   }
 
+  /** Get tags (key-value pairs) currently attached to this Sandbox from the server. */
+  async getTags(): Promise<Record<string, string>> {
+    try {
+      const resp = await client.sandboxTagsGet({ sandboxId: this.sandboxId });
+
+      const tags: Record<string, string> = {};
+      for (const tag of resp.tags) {
+        tags[tag.tagName] = tag.tagValue;
+      }
+      return tags;
```

Comment:
Very nit: Can this be done outside the `try` block? (I like to have less indented code 😅)